ocrd_segment/repair.py

```diff
@@ -87,49 +88,78 @@ def process(self):
                 self.sanitize_page(page, page_id)
                 
             #
-            # plausibilize region segmentation (remove redundant regions)
+            # plausibilize region segmentation (remove redundant text regions)
```

Comment:
I think we could do it for other region types as well! Separators and images tend to be fragmented as well.

Reply:
Good point. But:
1. the merged polygons could quickly encompass a much larger region, so one would have to check whether the combined exterior area is not much more than the sum of its parts (also true for text regions already)
2. we have to be careful which region types we allow to combine with each other. 

Possibilities:
- separator+separator: perhaps only vlines+vlines and hlines+hlines?
- image+image: this might have images creep over the whole page, e.g. above Tesseract thinks the darker margin is an image...
- table+table: unlikely (at least for Tesseract)
- table+text: ...
- separator+text: probably best left to clipping?
- image+text: probably best left to clipping?
- ...

Reply:
>     1. the merged polygons could quickly encompass a much larger region, so one would have to check whether the combined exterior area is not much more than the sum of its parts (also true for text regions already)

just added a check to avoid that problem